src/collections/Breadcrumb/BreadcrumbSection.js

```diff
@@ -0,0 +1,80 @@
+import React, { PropTypes } from 'react'
+import cx from 'classnames'
+import META from '../../utils/Meta'
+import { customPropTypes, useKeyOnly } from '../../utils/propUtils'
+
+/**
+ * A section sub-component for Breadcrumb component.
+ */
+function BreadcrumbSection(props) {
+  const {
+    active, children, className, link, href, onClick, ...rest,
+  } = props
+
+  const handleClick = (e) => {
+    if (onClick) onClick(e)
```

Comment:
Code coverage reports that this branch not tested. How I make it tested properly?